cmd/fluxctl/list_workloads_cmd.go

```diff
@@ -101,3 +107,16 @@ func policies(s v6.ControllerStatus) string {
 	sort.Strings(ps)
 	return strings.Join(ps, ",")
 }
+
+// Extract workloads having its container name equal to containerName
+func filterByContainerName(workloads []v6.ControllerStatus, containerName string) (filteredWorkloads []v6.ControllerStatus) {
+    for _, workload := range workloads {
+        if len(workload.Containers) > 0 {
+			c := workload.Containers[0]
```

Comment:
Why is only the first container being checked?

Reply:
Well that's a mistake :thinking: